clients/tabby-agent/src/AgentConfig.ts

```diff
@@ -0,0 +1,23 @@
+export type AgentConfig = {
+  server?: {
+    endpoint?: string;
+  };
+  logs?: {
+    level?: "debug" | "error" | "silent";
+  };
+  analytics?: {
+    enabled?: boolean;
```

Comment:
`enabled` -> `disabled`, means the anonymousUsageTracking is by default enabled.